import_table.py

```diff
@@ -1,88 +1,166 @@
-import records
+"""
+Imports the data from the CSV file generated by Qualtrics.
+Any entires already in the database are ignored during the import.
+"""
+import sys
 import csv
+import records
+import time
 
+# Be sure to check the Qualtrics forms for what these values should be.
+JUDGE_ROLE_NUM = '1'
+"""A string that represents the judge role in the volunteer form's CSV file."""
+MENTOR_ROLE_NUM = '2'
+"""A string that represents the mentor role in the volunteer form's CSV file."""
 
-'''
-- Pull Data from CSV file
-- For every entry, call the add_participant() method
-'''
-# IF: file is importing participants, it MUST have 'participant' in the name
-# ELSE: file is treated as containing leaders 
```

Comment:
I based the "attribute names used elsewhere in the bot code" off of the [`user_data` variable at line 76 of `web.py`](https://github.com/hackohio/ohio-discord-bot-2024/blob/9f785d84b70bf54347fcd9fe9e5687d6e58b0c91/web.py#L76-L87). To my knowledge, only the first name attribute is used for some of the bot messages.

I can't use the webhook attribute names (e.g., "firstName", "lastName", "classTeam", etc.) because they are on the second row of the CSV files.

So I included these two dictionaries to map the attribute names in the first row of the CSV files to the attribute names used in the linked code snippet.